CHANGELOG.md

```diff
@@ -124,9 +124,16 @@ regression test suite and patches at https://github.com/seqan/seqan3/tree/master
   * `seqan3::aminoacid_similarity_matrix::BLOSUM45` is replaced by `seqan3::aminoacid_similarity_matrix::blosum45`.
   * `seqan3::aminoacid_similarity_matrix::BLOSUM62` is replaced by `seqan3::aminoacid_similarity_matrix::blosum62`.
   * `seqan3::aminoacid_similarity_matrix::BLOSUM80` is replaced by `seqan3::aminoacid_similarity_matrix::blosum80`.
-* We deprecated seqan3::alignment_coordinate and it will be removed in 3.1.0
+* Deprecated seqan3::alignment_coordinate and it will be removed in 3.1.0
   ([\#2602](https://github.com/seqan/seqan3/pull/2602)).
 
+Header Changes:
+
+```cpp
+#include <seqan3/alignment/band/{all => static_band}.hpp>
+#include <seqan3/alignment/matrix/{alignment_coordinate => detail/advanceable_alignment_coordinate}.hpp>
+#include <seqan3/alignment/scoring/gap_scheme.hpp> [deleted without replacement]
```

Comment:
```suggestion
#include <seqan3/alignment/band/static_band.hpp> [deleted without replacement]
#include <seqan3/alignment/matrix/alignment_coordinate.hpp> [deleted without replacement]
#include <seqan3/alignment/scoring/gap_scheme.hpp> [deleted without replacement]
```

* detail code shouldn't be advertised.
* gap_scheme was partly deprecated in 3.0.2